aspnetcore/fundamentals/servers/kestrel.md

```diff
@@ -14,17 +14,25 @@ By [Tom Dykstra](https://github.com/tdykstra), [Chris Ross](https://github.com/T
 
 :::moniker range=">= aspnetcore-8.0"
 
-Kestrel is a cross-platform [web server for ASP.NET Core](xref:fundamentals/servers/index). Kestrel is the web server that's included and enabled by default in ASP.NET Core project templates.
+Kestrel is a cross-platform [web server for ASP.NET Core](xref:fundamentals/servers/index). Kestrel is the recommended server for ASP.NET Core, and it's configured by default in ASP.NET Core project templates.
 
-Kestrel supports the following scenarios:
+Some of the best features of ASP.NET Core Kestrel server include:
```

Comment:
"best" sounds like marketing language
```suggestion
Some of the features of ASP.NET Core Kestrel server include:
```

Reply:
Agreed, if you must use an adjective, notable maybe. But best to just drop best.